chore: add `remote` company requirements (#342)
marmelo/tech-companies-in-portugal: CONTRIBUTING.md

Some additional things to keep in mind:
* The company must allow working in/from Portugal;

>   * Fully remove companies might also fit here if either is true:
>     * Career page explicitly mentions that it hires remote people from PT / EU;
>     * There are PT engineers already working remotely for said company;

Reviewer: ```suggestion
  * Fully remote companies might also fit here if either is true:
    * Careers page explicitly mentions that the company hires remote people from PT or EU;
    * There are PT engineers already working remotely for said company.
```
